### ISSUE TITLE
Switch bots collecting mass in game with button press.

### ISSUE THREAD

**Issue by boboftw** [davidmann4/ogar-feeder-bot#437]
I saw on some servers bots that are collecting mass and then go in the bot owner.
I wonder if some of you guys know how to do it.
I know that you can change from config.js from "default to blind" , but i need to do this IN-GAME with a button press.

I hope someone can help me , would be nice , thanks.

**Comment by ghost** [comment]
Raga.pw have this, more bots come in one local get mass and go the player

**Comment by boboftw** [comment]
Yes , but i saw players using this too , so someone should know how to do that.

**Comment by David1ali12** [comment]
no one share this code

**Comment by hassanila** [comment]
Its too easy that's why noone shares it :D

**Comment by boboftw** [comment]
I just want some help , dont come here and say "no one share"/" is impossible" , etc.
Here is my skype: albert.unik
Also sorry for bad eng.

**Comment by boboftw** [comment]
Also , hassan if is "too easy" why don't you tell us? (rofl)

**Comment by SrNicolas** [comment]
Got It, ez

**Comment by David1ali12** [comment]
XD

**Comment by rafailrafail** [comment]
https://youtu.be/YP4LR7wr4DY

**Comment by boboftw** [comment]
Still searching someone who want to help :+1:

**Comment by CupidStunts** [comment]
me too....cmon someone be a legend & help us out lol

**Comment by wartoz** [comment]
we need this option

**Comment by CupidStunts** [comment]
hassanila97,SrNicolas if its so easy why dont all bot companys have it huh? if you guys ever make a bot company lol i sure wont buy or point anyone your way haha....oh & btw ive tried adding it but it dont work :(

**Comment by Martinali12** [comment]
shut the fuck up @Wartoz  @CupidStunts  ppl llike you get our codes then make shityybot,tk and then share for ppl and then make alot of lag on agar.io WTF FWBOT.GA fucked agar.io with lag #stop making lag

**Comment by ravyn120** [comment]
I'll tell you guys for a few fb cookies

**Comment by SrNicolas** [comment]
I already have almost done my selling company so I'm good :dancer:
